docs/services/ml/_index.md

```diff
@@ -61,45 +73,3 @@ Viam supports the following model frameworks:
 - [ONNX](https://onnx.ai/): with the [`onnx_cpu` ML model service](https://github.com/viam-labs/onnx-cpu)
 
 For more information, see [Model framework support](/services/ml/upload-model/#model-framework-support).
-
-## Use machine learning with your machine
-
-<table>
```

Comment:
My thinking is that this is now in how-tos so it's totally duplicative

Reply:
I'm fine leaving it here if you like--I deleted it because I thought we are meant to be de-duplicating content that has migrated to how-tos, but I can see that if we leave it here too it'll be more easily discoverable

Reply:
you cannot assume that people read all of the page, please leave it in

Reply:
Sorry I don't follow. Meaning you can't assume people read the how-to page?